documentation/de/front_matter/copyright_and_attributions/index.html

```diff
@@ -11,6 +11,10 @@
     <title>Copyright :: Dokumentation für Selenium</title>
```

Comment:
Hi @boris779 
This can be the same as other pages as copyright and attributions.

Reply:
You mean to keep that in English?

Reply:
It just says copyright. But in remaining Pages it is like copyright and attributions

Reply:
Ok. 
Attributions will be translated with "Zuwendungen", but it is a very uncommon word in German. I would suggest to translate it with "Unterstützer"  which means supporter or facilitator.

I translated the whole page and pushed it into that pull request

Reply:
oh, it makes sense. Thank you for the clarification.